### ISSUE TITLE
Multiple HTTP sources not working

### ISSUE THREAD

**Issue by Dazza0311** [ytti/oxidized#3185]
I have edited my config file to source from 2 different Libre systems, I however see that it can only do one.
I am not sure if this has something to do with Puma starting in single mode.

Below is my current config;

![image](https://github.com/ytti/oxidized/assets/170534812/5c15a25e-2116-4ba7-8ad3-3282e273b838)

**Comment by Dazza0311** [comment]
@ytti please can you assist

**Comment by Dazza0311** [comment]
For some reason it only pulls the devices from the last source and not the first source

**Comment by ytti** [comment]
Multiple sources are not supported, it wouldn't be particularly hard to add multiple source support, but it couldn't possibly work in suggested config, because you overwrite the 'http' key, it would need to be something like:

```yaml
source:
   - http:
        url: url1
   - http:
       url: url2
```

**Comment by Dazza0311** [comment]
Could your please provide me with an example syntax as I need to input 2x Auth tokens. Not sure how I should be doing this

**Comment by Dazza0311** [comment]
@ytti  I have edited it with the below config

![image](https://github.com/ytti/oxidized/assets/170534812/c53ba85d-2fd7-4945-bea3-92c008a47c3d)

I get this error when trying to run oxidized

![image](https://github.com/ytti/oxidized/assets/170534812/d923d020-ae6a-402a-a360-2c3883b9b3a7)

**Comment by randybb** [comment]
ytti mentioned that multiple sources are not supported and you are still trying. Even it would be, your yaml syntax is not valid - you cannot have the same keys + he mentioned how it could be written with `source:` as an array (note the dashes), but still this is not supported.

**Comment by Dazza0311** [comment]
Is there a way to get support for this?
Can this be a requested development?

**Comment by ytti** [comment]
Yes it would be possible to add such support, but I'm at least not incentivized to work on this, perhaps someone else is.

**Comment by jseifeddine** [comment]
+1

**Comment by leec-666** [comment]
I handled this with a little Flask proxy running on the same host as Oxidized that queries all sources (Netbox devices and VMs in my case) and returns the combined result.
